Automattic/wp-calypso
# Woo: Update edit state on category create

This adds reducer code to do the following upon successful product
category create:

1. Replace category id placeholders for the real id in any product edits.
2. Remove the category from creates.

To test: `npm test`
Additional optional testing: Create a new category and observe the redux edit state in dev tools afterwards. The category should be gone from creates, and the product category reference should have a real numeric id in it.
